src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images\star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars_star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--1 .stars_star:nth-child(-n + 1),
```

Comment:
think using this selectors should be enough `.stars--1 :nth-child(-n + 1)`,
`.stars--2 :nth-child(-n + 1)` etc..